configs/supervisord.conf

```diff
@@ -10,14 +10,14 @@ stdout_logfile_maxbytes=0
 
 [program:cron]
 command=/usr/sbin/cron -f -L 15
-stderr_logfile=/var/log/supervisor/cron_error.log
+stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
-stdout_logfile=/var/log/supervisor/cron.log
+stdout_logfile=/dev/stderr
 stdout_logfile_maxbytes=0
 
 [program:c-bastion]
 command=python /usr/local/bin/c-bastiond
-stderr_logfile=/var/log/supervisor/c-bastion_error.log
+stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
-stdout_logfile=/var/log/supervisor/c-bastion.log
```

Comment:
In the stdout of the Docker container as per our separation of concern design. Log shipping is the job of the Docker host, not the container.